environment.py

```diff
@@ -154,6 +154,16 @@ def env() -> Mapping[str, Optional[str]]:
         # deployment. This will become the value of the Owner tag on all resources.
         'AZUL_OWNER': None,
 
+        # An email address to subscribe to the SNS topic for monitoring
+        # notifications in the current deployment. Warn all members of the group
+        # to ignore any "Subscription Confirmation" emails and not to click the
+        # "Confirm subscription" link contained within, since doing so would
+        # confirm the subscription in a way that allows anyone with an
+        # unsubscribe link to unsubscribe the entire group from the topic.
+        # Instead, confirmation of the subscription should be done when prompted
+        # to do so during the `make deploy` process.
+        'AZUL_NOTIFICATION_EMAIL': None,
```

Comment:
```suggestion
        'AZUL_MONITORING_EMAIL': None,
```

Reply:
This was from a commit in https://github.com/DataBiosphere/azul/pull/4194 .
The PR for #4194 had been merged into `develop` and this PR was rebased on top, however then the merge was reverted which left extra commits in this PR. I will fix this with my next push of this PR to GitHub.

Reply:
Feel free to fix these things proactively while the PR is in Review requested.